nni/runtime/msg_dispatcher.py

```diff
@@ -87,6 +87,7 @@ def save_checkpoint(self):
     def handle_initialize(self, data):
         """Data is search space
         """
+        _logger.info('Initial search space: %s, %s', type(data), data)
```

Comment:
If the level is INFO, it should not contain `type(data)`.

Reply:
sure, removed